### PR TITLE
Feature-add-more-featured-photos (prod)

### PR DESCRIPTION
**Pull request by cgettings**
PR checklist:
- [x] Changes previewed and QC'ed
- [x] In PR, mention team member for FYI, review, or approval

@mmontesanonyc, @etorem I decided to add unique featured images to a few data stories. I also modified the `photocredit` field, mostly to add "City of New York", as stipulated in the usage guidelines on the Mayor's Office of Photography site.

### CODE REVIEW

**Comment by mmontesanonyc** [comment]
Good catch on the `photocredit`. 
@etorem can you put a set of eyes on the new images that Chris added? Approval is your call. Thanks!

File: content/data-stories/hia/171109_dailylife0059.jpg
Comment (cgettings) [inline content/data-stories/hia/171109_dailylife0059.jpg:1]: @etorem This is in the Bronx, standing in Pelham - Throgs Neck (a neighborhood outside the top 10 for PM 2.5 but in the top 10 for health effects) toward Hunts Point - Mott Haven (a neighborhood in the top 10 of both).

File: content/data-stories/air-quality-by-neighborhood/170801_NNO_Bklyn096DSC_114.jpg
Comment (cgettings) [inline content/data-stories/air-quality-by-neighborhood/170801_NNO_Bklyn096DSC_114.jpg:1]: @etorem I think this one has a strong "outer-boro air quality" vibe.

**Comment by etorem** [comment]
These look really nice!

File: content/data-stories/urban-heat-island/Harlem (1).jpg
Comment (cgettings) [inline content/data-stories/urban-heat-island/Harlem (1).jpg:1]: @etorem This is in Harlem, looking north up 12th Ave just south of MLK Blvd. Harlem has lots of points on the map in the data story, and this block has very few trees and lots of blacktop, so I thought it was close to perfect.

Comment (etorem) [reply content/data-stories/urban-heat-island/Harlem (1).jpg:1]: @cgettings it is indeed perfect!

**Pull request merged**